omnibus/config/software/chef-workstation-app.rb

```diff
@@ -0,0 +1,53 @@
+name "chef-workstation-app"
+license "Apache-2.0"
+skip_transitive_dependency_licensing
+license_file "LICENSE"
+
+source git: "https://github.com/chef/chef-workstation-tray"
+default_version "SHACK-322/omnibusify-chef-workstation-app"
```

Comment:
Is this going to become a PR or do we need a long running branch?

Reply:
That  PR has merged today, I'll update to master.